.gitattributes

```
@@ -60,6 +60,7 @@ Include/token.h             linguist-generated=true
 Lib/token.py                linguist-generated=true
 Parser/token.c              linguist-generated=true
 Programs/test_frozenmain.h  linguist-generated=true
+configure                   linguist-generated=true
```

``aclocal.m4`` is also auto-generated

```suggestion
configure                   linguist-generated=true
aclocal.m4                  linguist-generated=true
```

---

Adressed. I didn't work with autoconf before so I hope I missed nothing more.